custom_components/myskoda/coordinator.py

```diff
@@ -79,8 +80,12 @@ def __init__(
         myskoda.subscribe(self._on_mqtt_event)
 
     async def _async_update_data(self) -> State:
-        vehicle = await self.myskoda.get_vehicle(self.vin)
-        user = await self.myskoda.get_user()
+        _LOGGER.debug("Performing scheduled update of all data for vin %s", self.vin)
+        try:
+            vehicle = await self.myskoda.get_vehicle(self.vin)
+            user = await self.myskoda.get_user()
+        except ClientError as err:
+            raise UpdateFailed("Error getting update from MySkoda API: %s", err)
```

Comment:
I think it's pylint (which I use at work) which always yells about me about raising `from` the original exception.

```suggestion
            raise UpdateFailed("Error getting update from MySkoda API: %s", err) from err
```

Reply:
Was just following the examples about calling UpdateFailed and it passed the linter 😂

Reply:
Yeah it's not a must for me. It's this one: https://pylint.readthedocs.io/en/stable/user_guide/messages/warning/raise-missing-from.html